app/components/elements/TextInput.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box } from 'rebass/styled-components';
+import { Label, Input } from '@rebass/forms';
+import { Caption } from './FontStyles';
+import { fonts, fontSizes, colors, borders, radii, space } from '../../themes/baseTheme';
+
+export const TextInput = props => {
+  const StyledInput = styled(Input)`
```

Comment:
It's important, for performance reasons, to define the styled component outside of the render method or functional component that will be using it.

See https://styled-components.com/docs/basics#define-styled-components-outside-of-the-render-method for reference

Reply:
Should we also change it in `IconButton` then? (I used this as a reference to build this component out - https://github.com/tidepool-org/blip/blob/web-559-component-library/app/components/elements/IconButton.js#L8)

Reply:
Ah yes!  I'll change it there.  Thanks for pointing that out!